.github/workflows/slither.yml

```diff
@@ -0,0 +1,39 @@
+name: 'Slither Analysis'
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - develop
+
+jobs:
+  slither-analyze:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      # Workaround to prevent slither-action from trying to install JS deps.
+      # Without this step, it detects the `package.json``, and since there is no
+      # lockfile it defaults `npm install` which fails due to the preinstall
+      # script to enforce pnpm. https://github.com/crytic/slither-action/issues/44#issuecomment-1338183656
+      - name: Remove package.json
+        run: rm packages/contracts-bedrock/package.json
+
+      - name: Run Slither
+        uses: crytic/slither-action@v0.3.1
```

Comment:
This will use the latest foundry nightly instead of our pinned version, which I think is ok for now since it shouldn't affect slither results. Can switch to ci builder image in a future PR